dart/collision/bullet/BulletCollisionNode.cpp

```diff
@@ -55,9 +55,11 @@ namespace collision {
 BulletCollisionNode::BulletCollisionNode(dynamics::BodyNode* _bodyNode)
   : CollisionNode(_bodyNode)
 {
-  for (size_t i = 0; i < _bodyNode->getNumCollisionShapes(); i++)
+  auto collShapeNodes = _bodyNode->getShapeNodes<dynamics::CollisionAddon>();
```

Comment:
I wonder if we should name this function ``getShapeNodesWith<T>()`` or something like that to make it more clear that it's a filter.